clients/stellar-relay-lib/Cargo.toml

```diff
@@ -28,6 +28,8 @@ serde = { version = "1.0.152", features = ["alloc"] }
 serde_json = "1.0.93"
 serde_with = "2.2.0"
 
+wallet = { path = "../wallet" }
```

Comment:
If I were to choose between duplicate codes vs adding another lib dependency, I would prefer duplicate code.

However it looks like we're just using it in the examples and test cases. We can transfer this line under `dev-dependencies`

Reply:
Moved!